### PR TITLE
Added home page

### PR DESCRIPTION
**Pull request by anshikaa12**
> Added nav , footer and main section
>Not responsive
>haven't added router yet!(Hence used a tags for now)
ecomm_homeReact

### CODE REVIEW

**Review by aqsaaqeel — APPROVED** [review]
Looks good. You can host it on netlify so its easier to see the preview.

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+function Footer() {
+    return (
+        <div>
+            <section class="footer">
```
Comment (ArjunGTX) [inline src/components/Footer.jsx:6]: Replace all occurances of `class` with `className`

File: src/components/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+
+
+function Nav() {
+    return (
+        <div>
+            <div class="nav">
+                <div class="first-nav">
+                    <img src="https://res.cloudinary.com/de5it79dw/image/upload/v1648491301/Pixelion_2_-PhotoRoom_q7mfmo.png" alt="" />
```
Comment (ArjunGTX) [inline src/components/Nav.jsx:9]: Give a meaningful `alt` for images

File: src/css/base/base.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    scroll-behavior: smooth;
+}
+
+:root {
+    --primary-color: #008e76;
+    --text-color: #0c080c;
+    --accent-color: #af7a6d;
+    --bg-color: #E6EAED;
+    --alt-text-color: #36383C;
+    --complement-btn-text-color: white;
```
Comment (ArjunGTX) [inline src/css/base/base.css]: You can give this color as `#ffffff` to keep it consistent

File: src/css/base/base.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+.hide {
+    display: none;
+}
+
+@media only screen and (max-width:550px) {
+
+    html {
+        font-size: 37.5%;
+        /*6px*/
```
Comment (ArjunGTX) [inline src/css/base/base.css]: Remove commented code

File: src/css/base/base.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    }
+}
+
+@media only screen and (max-width:830px) {
+    html {
+        font-size: 43.75%;
+        /*7px*/
```
Comment (ArjunGTX) [inline src/css/base/base.css]: Same as above

**Review by ArjunGTX — APPROVED** [review]

**Pull request merged**